package.json

```diff
@@ -127,7 +127,6 @@
     "build": "NODE_ENV=production webpack --config web/webpack.prod.config.js --mode 'production'",
     "build-clean": "yarn run clean && yarn run build",
     "clean": "rm -rf ./web/public/build/",
-    "develop": "./node_modules/.bin/concurrently --kill-others-on-fail -n flow,webpack,translations,mypy 'yarn flow --continuous_scroll' 'yarn webpack' 'yarn cli-translations watch' 'yarn mypy --continuous_scroll'",
     "flow-check": "echo skipping flow check",
```

Comment:
I killed the whole line since I assumed it was broken!

- The yarn mypy entry was missing
- flow doesn't run on ubuntu arm
- cli-translations entry was missing

You want me to fix mypy and translations as part of this PR then?

Reply:
Develop actually is broken lol, nobody uses it because it crashes super frequently and is a nuisance. 

So I would:
- Remove develop
- Add in `cli-translations` - done. My reasoning on including this change in this diff is translations has some dependencies like watchman that are affected by these overall changes so I want to ensure it works.
- Keep `translations` - done
- Remove `flow-check` since it doesn't do anything and won't work with ubuntu
- No opinion on mypy, but seems like you got that working


More broadly and not in the scope of this diff, I think we should add back in the formatting, linting, and flow code that was dropped from our repo. It's annoying to need to manually format code when working in Harmony and lint, flow, and mypy will just help us verify Harmony works and catch errors.